src/stories/Drawer.stories.js

```diff
@@ -0,0 +1,96 @@
+import BasicDrawer from '../components/drawer/BasicDrawer.vue'
+
+// This is required for each story
+export default { title: 'Drawer' }
+
+// Customize components here.
+export const basicDrawer = () => ({
+  components: { BasicDrawer },
+  data () {
+    return {
+      drawer: false,
+      direction: 'rtl',
+      title: 'I am the title'
+    }
+  },
+  watch: {
+    drawer (newValue, oldValue) {
+      if (!newValue) {
+        this.drawer = false
+      }
+    }
+  },
```

Comment:
This code is supposed to be in the `BasicDrawer` component not outside.:sweat_smile: